### PR TITLE
Resolve python 3 issues

### PR DESCRIPTION
**Pull request by codingjoe**
I actually threw in a little more than that. I more or less changed the the test design. Let me know if that is interesting or not and I can prepare a better pull request.

### CODE REVIEW

File: health_check_storage/base.py [hunk 1]
```diff
@@ -20,7 +21,7 @@ class MyStorageHealthCheck(StorageHealthCheck):
     storage = None
 
     def get_storage(self):
-        if isinstance(self.storage, basestring):
+        if isinstance(self.storage, text_type):
```
Comment (syphar) [inline health_check_storage/base.py]: it's `six.string_types`

Comment (syphar) [inline health_check_storage/base.py]: @codingjoe cc

Comment (codingjoe) [inline health_check_storage/base.py]: I did not want to add an additional dependency and six shipped with django, hence the import location.

Comment (codingjoe) [inline health_check_storage/base.py]: Oh wait, I got it now. Sure thing I'll fix it.

**Comment by codingjoe** [comment]
Project seems dead. Everyone who's looking for a working fork: https://github.com/Thermondo/django-health-check

**Pull request closed without merging**